client/cpu9p_windows.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"errors"
+	"os"
+
+	"github.com/hugelgupf/p9/p9"
+)
+
+// GetAttr implements p9.File.GetAttr.
+//
+// Not fully implemented.
+func (l *CPU9P) GetAttr(req p9.AttrMask) (p9.QID, p9.AttrMask, p9.Attr, error) {
+	qid, fi, err := l.info()
+	if err != nil {
+		return qid, p9.AttrMask{}, p9.Attr{}, err
+	}
+
+	t := uint64(fi.ModTime().Unix())
+	// In the unix-style code, we derive the Mode directly
+	// from the syscall.Stat_t. That is not available on
+	// Windows, so we have to reconstruct some mode bits
+	// in other ways.
+	mode := p9.FileMode(fi.Mode())
+	if fi.IsDir() {
+		mode |= p9.ModeDirectory
+	}
+	attr := p9.Attr{
+		Mode:             mode,
+		UID:              0,
+		GID:              0,
+		NLink:            3,
+		Size:             uint64(fi.Size()),
+		BlockSize:        uint64(512),
+		Blocks:           uint64(fi.Size() / 512),
+		ATimeSeconds:     t,
+		ATimeNanoSeconds: 0,
+		MTimeSeconds:     t,
+		MTimeNanoSeconds: 0,
+		CTimeSeconds:     t,
+		CTimeNanoSeconds: 0,
+	}
+	valid := p9.AttrMask{
+		Mode:   true,
+		UID:    true,
+		GID:    true,
+		NLink:  true,
+		RDev:   true,
+		Size:   true,
+		Blocks: true,
+		ATime:  true,
```

Comment:
It seems like `ATime` stands for "Access Time" and we're setting it to `ModTime` which is "Modified Time" which seems incorrect. Is it possible to just set this AttrMask value to false to indicate we don't know the `ATime`?